### PR TITLE
[Term Entry] Kotlin Math .cos()

### PR DESCRIPTION
**Pull request by yangvianno**
### Description

Added new term entry on the .cos() method for Kotlin: Math

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/cosin/cosin.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`math.cos()`** method in Kotlin's `math` class is used to calculate the cosine of an angle 
+in radians.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/cosin/cosin.md:17]:
```suggestion
The **`math.cos()`** method in Kotlin's `math` class is used to calculate the cosine of an angle in radians.
```

File: content/kotlin/concepts/math-methods/terms/cosin/cosin.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Syntax
+
+```pseudo
+import kotlin.math.cos
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/cosin/cosin.md:22]:
```suggestion
```
no need to mention the lib here

File: content/kotlin/concepts/math-methods/terms/cosin/cosin.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+val cosineValue = cos(angle)
+```
+
+- `angle`: The angle in radians for which you want to calculate the cosine.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/cosin/cosin.md:26]:
```suggestion
- `angle`: The angle in radians for which you want to calculate the `cosine`.
```

File: content/kotlin/concepts/math-methods/terms/cosin/cosin.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+The `math.cos()` method returns a value of type Double representing the cosine of the 
+input angle.
+
+## Example
+
+The example below demonstrates how to use `math.cos()` method to calculate the cosine of 
+an angle.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/cosin/cosin.md:34]:
```suggestion
The example below demonstrates how to use `math.cos()` method to calculate the cosine of an angle:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few comments for now

File: content/kotlin/concepts/math-methods/terms/cosin/cosin.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+```pseudo
+import kotlin.math.cos
+val cosineValue = cos(angle)
+```
+
+- `angle`: The angle in radians for which you want to calculate the cosine.
+
+The `math.cos()` method returns a value of type Double representing the cosine of the 
+input angle.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/cosin/cosin.md:29]:
```suggestion
- The `math.cos()` method returns a value of type Double representing the cosine of the input angle.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]

**Review by SSwiniarski — APPROVED** [review]
@yangvianno I fixed the name of the .md file and made a small change to eliminate a second-person pronoun. Now it looks good. @dakshdeepHERE can you approve this so we can merge.

**Review by dakshdeepHERE — APPROVED** [review]

**Comment by dakshdeepHERE** [comment]
Done @SSwiniarski

**Pull request merged**